package.json

```diff
@@ -1,11 +1,19 @@
 {
-  "name": "clickhouse-js",
+  "name": "@clickhouse/client",
   "version": "0.0.1",
-  "description": "The official js client for the ClickHouse DB",
-  "main": "index.js",
+  "description": "Official JS client for ClickHouse DB",
+  "license": "Apache-2.0",
+  "keywords": [
+    "clickhouse",
+    "sql",
+    "client"
+  ],
+  "engines": {
+    "node": ">=14"
+  },
   "scripts": {
-    "build": "rm -rf dist; webpack --mode=development",
-    "build:all": "tsc --project tsconfig.dev.json",
+    "build": "rm -rf dist; tsc",
+    "build:all": "rm -rf dist; tsc --project tsconfig.dev.json",
```

Comment:
I just realized we CONTRIBUTING.md doesn't cover how to build and run a type check